### PR TITLE
Slight method refactoring

### PR DESCRIPTION
**Pull request by lloydjatkinson**
Removed ugly and messy if statements. 
Made colour lookup for use in future methods. 
Removed violations of DRY.

### CODE REVIEW

File: EmbedTool/EmbedTool.cs [hunk 1]
```diff
@@ -10,8 +10,28 @@ namespace EmbedsOnFly
 {
     public class EmbedTool : ModuleBase
     {
-        EmbedBuilder basic;
-        EmbedFooterBuilder footer;
+        private EmbedBuilder basic;
+        private EmbedFooterBuilder footer;
+
+        private readonly IDictionary<string, Color> _paletteMap = new Dictionary<string, Color>()
+        {
```
Comment (TAGC) [inline EmbedTool/EmbedTool.cs:17]: You can use the cleaner dictionary initialisation syntax here if you prefer:

```cs
{
    ["aqua"] = new Color(0, 255, 255),
    ["black"] = new Color(0, 0, 0),
    ...
}
```

File: EmbedTool/EmbedTool.cs [hunk 2]
```diff
@@ -10,8 +10,28 @@ namespace EmbedsOnFly
 {
     public class EmbedTool : ModuleBase
     {
-        EmbedBuilder basic;
-        EmbedFooterBuilder footer;
+        private EmbedBuilder basic;
+        private EmbedFooterBuilder footer;
+
+        private readonly IDictionary<string, Color> _paletteMap = new Dictionary<string, Color>()
```
Comment (TAGC) [inline EmbedTool/EmbedTool.cs:16]: I'd make this static to avoid re-instantiation of the dictionary on each instantiation of `EmbedTool`.

Comment (lloydjatkinson) [reply EmbedTool/EmbedTool.cs:16]: Agreed.

**Comment by lloydjatkinson** [comment]
@MCUnderground feel free to accept this PR if you feel if it's of value.

**Pull request closed without merging**